apps/test/unit/code-studio/pd/workshop_dashboard/components/local_summer_workshop_survey_resultsTest.js

```diff
@@ -0,0 +1,117 @@
+import {shallow, mount} from 'enzyme';
+import {expect} from 'chai';
+import sinon from 'sinon';
+import React from 'react';
+
+describe("Local Summer Workshop Management", () => {
+  const getFakeSurveyDataForFacilitator = () => {
+    return {
+      facilitator_breakdown: false,
+      this_workshop: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2'],
+        'num_enrollments': '10',
+        'num_surveys': '8',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5'
+      },
+      all_my_local_workshops: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2', 'Feedback 3', 'Feedback 4'],
+        'num_enrollments': '200',
+        'num_surveys': '180',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5'
+      }
+    };
+  };
+
+  const getFakeSurveyDataForWorkshopOrganizer = () => {
+    return {
+      facilitator_breakdown: true,
+      facilitator_names: ['Han', 'Chewbacca'],
+      this_workshop: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2'],
+        'num_enrollments': '10',
+        'num_surveys': '8',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5',
+        'how_clearly_presented': {'Han': 5, 'Chewbacca': 1},
+        'things_facilitator_did_well': {'Han': ['Shot first'], 'Chewbacca': ['Roared a lot']},
+        'facilitator_names': ['Han', 'Chewbacca']
+      },
+      all_my_local_workshops: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2', 'Feedback 3', 'Feedback 4'],
+        'num_enrollments': '10',
+        'num_surveys': '8',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5',
+        'how_clearly_presented': 5
+      }
+    };
+  };
+
+  describe("View summarization", () => {
+    it("Spinner displayed initially", () => {
+      let localSummerWorkshopSurveyResults = shallow(
+        <LocalSummerWorkshopSurveyResults
+          params={{workshopId: '1', facilitators: []}}
+        />
+      );
+
+      expect(localSummerWorkshopSurveyResults.state('loading')).to.be.true;
+      expect(localSummerWorkshopSurveyResults.find('Spinner')).to.have.length(1);
+    });
+
+    it("Table gets displayed after loading is completed", () => {
+      let server = sinon.fakeServer.create();
+      server.respondWith(
+        'GET',
+        '/api/v1/pd/workshops/1/local_workshop_survey_report',
+        [200, {"Content-Type": "application/json"}, JSON.stringify(getFakeSurveyDataForFacilitator())]
+      );
+
+      let localSummerWorkshopSurveyResults = mount(
+        <LocalSummerWorkshopSurveyResults
+          params={{workshopId: '1'}}
+        />
+      );
+
+      server.respond();
+
+      expect(server.requests.length).to.equal(1);
+      expect(localSummerWorkshopSurveyResults.state('loading')).to.be.false;
+      expect(localSummerWorkshopSurveyResults.find('table')).to.have.length(1);
+      expect(localSummerWorkshopSurveyResults.find('table th')).to.have.length(3);
```

Comment:
This is fine, and more concise in this context anyway, but note you can also use the returned wrapper from `find` to further filter:

```
const table = localSummerWorkshopSurveyResults.find('table');
expect(table).to.have.length(1);
expect(table.find('th')).to.have.length(3);
```